brave-intl/publishers
# Improve User Funds Only front end

## Improve User Funds Only front end
![image](https://user-images.githubusercontent.com/5459225/60199573-dc520080-9809-11e9-97fc-9c54336c8865.png)


<img width="1285" alt="Screen Shot 2019-06-26 at 11 32 27 AM" src="https://user-images.githubusercontent.com/5459225/60199539-cba18a80-9809-11e9-93bf-b4c68aa4c050.png">
<img width="1198" alt="Screen Shot 2019-06-26 at 11 32 10 AM" src="https://user-images.githubusercontent.com/5459225/60199534-c9d7c700-9809-11e9-8e52-e7dbacec6e1a.png">

#### Features

- Interacts with the Promo server to put publishers in the correct state once 
- Only shows contributions for users who are in the "user_funds_only" state
- Show warning in the promo registrations 
- Creates partials for some of the panels on `home.html.slim` page (It was getting pretty large)

#### How To Test

1. Create a new user, activate the promo registrations
2. Add a few channels
3. Move the user to "User funds only"
4. See that the dashboard has changed
